cms/djangoapps/contentstore/utils.py

```diff
@@ -160,7 +160,10 @@ def get_lms_link_for_certificate_web_view(user_id, course_key, mode):
 
 def course_image_url(course):
     """Returns the image url for the course."""
-    loc = StaticContent.compute_location(course.location.course_key, course.course_image)
+    try:
+        loc = StaticContent.compute_location(course.location.course_key, course.course_image)
+    except InvalidKeyError:
+        return None
```

Comment:
@Shrhawk I think you were working on one of your PR related to `course_image` and it was  from `lms` side? Can mention that here?